packages/stark-ui/src/modules/date-picker/components/date-picker.component.ts

```diff
@@ -391,6 +391,10 @@ export class StarkDatePickerComponent extends AbstractStarkUiComponent
 		}
 
 		if (changes["max"] || changes["min"] || changes["required"]) {
+			if (changes["required"]) {
```

Comment:
Personally I would put this in a setter or not nest it, but it's not a blocking issue for me.